rasa/train.py

```diff
@@ -348,6 +378,7 @@ async def _train_core_with_validated_data(
     train_path: Optional[Text] = None,
     fixed_model_name: Optional[Text] = None,
     additional_arguments: Optional[Dict] = None,
+    interpreter: Optional[Interpreter] = None,
```

Comment:
optional: looks nicer if additional arguments come after all other arguments
```suggestion
    interpreter: Optional[Interpreter] = None,
    additional_arguments: Optional[Dict] = None,
```

Reply:
I don't want to change the order in case somebody is passing everything as arguments (not as key word argument) and the changed order might break things for them. `train_core` is somewhat part of the public Python interface)